website/server/controllers/api-v3/tasks.js

```diff
@@ -41,7 +41,103 @@ let requiredGroupFields = '_id leader tasksOrder name';
  * @apiName CreateUserTasks
  * @apiGroup Task
  *
+ * @apiParam (Body) {string} text The text to be displayed for the task
+ * @apiParam (Body) {string} type Task type, options are: "habit", "daily", "todo", "reward".
+ * @apiParam (Body) {string[]} [tags] Array of UUIDs of tags
+ * @apiParam (Body) {string} [alias] Alias to assign to task
+ * @apiParam (Body) {string} [attribute] User's attribute to use, options are: "str", "int", "per", "con"
+ * @apiParam (Body) {boolean} [collapseChecklist=false] Determines if a checklist will be displayed
+ * @apiParam (Body) {string} [notes] Extra notes 
+ * @apiParam (Body) {number} [priority=1] Difficulty, options are 0.1, 1, 1.5, 2; eqivalent of Trivial, Easy, Medium, Hard.
+ * @apiParam (Body) {string[]} [reminders] Array of reminder data including a UUID, startDate and time
+ * @apiParam (Body) {string} [reminders.id] UUID of the reminder
```

Comment:
This seems a little odd. You won't have an array of just a reminder.id. I think it would be better to show `[{id: 'example'}]` or something to that effect.